.github/workflows/pr-build.yml

```
@@ -37,9 +66,41 @@ jobs:
       - name: Build with Flutter
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: flutter build apk --debug
+        run: |
+          flutter build apk --${{ inputs.app-variant }};
+      - name: Set env
+        run: |
+          echo "COMMIT_HASH=$(git rev-parse --short HEAD)" >> $GITHUB_ENV
       - name: Upload build
         uses: actions/upload-artifact@v3
         with:
-          name: revanced-manager
-          path: build/app/outputs/flutter-apk/app-debug.apk
+          name: revanced-manager-${{ env.COMMIT_HASH }}-${{ inputs.pr-number }}-${{ inputs.app-variant }}-${{ inputs.flutter-variant }}
+          path: build/app/outputs/flutter-apk/app-${{ inputs.app-variant }}.apk
```

TODO: include SHA hash

---

SHA isn't really necessary

---

Eh, it's easy to do, so I figure why not add it

```suggestion
          path: | 
            build/app/outputs/flutter-apk/app-${{ inputs.app-variant }}.apk
            build/app/outputs/flutter-apk/app-${{ inputs.app-variant }}.apk.sha1
```